Drivers/Periphs/Src/GPS.c

```diff
@@ -0,0 +1,63 @@
+#include "GPS.h"
+
+static QueueHandle_t *GPSRxQueue;
```

Comment:
I think this needs to be of type "QueueHandle_t" and not "QueueHandle_t *" since xQueueCreate is putting a queue handle value here.